bedrock/firefox/templates/firefox/content-blocking-tour/index.html

```diff
@@ -1,152 +0,0 @@
-
-{# "noindex" pages should not have the canonical or hreflang tags: bug 1442331 #}
-{% block canonical_urls %}<meta name="robots" content="noindex,follow">{% endblock %}
-
-{#- This will appear as <meta property="og:title"> which can be used for social share -#}
-{% block page_og_title %}
-  {{ _('Choose the independent browser') }}
-{% endblock %}
-
-{#- This will appear as <meta property="og:description"> which can be used for social share -#}
-{% block page_og_desc %}
-  {{ _('Firefox is non-profit, non-corporate, non-compromised. Choosing Firefox isn’t just choosing a browser. It’s a vote for personal freedom.') }}
-{% endblock %}
-
-{#- Override <meta property="og:url"> for social share -#}
-{% block page_og_url %}{{ url('firefox') }}{% endblock %}
-
-{% block string_data %}
-data-panel1-title="{{ _('New in Firefox: Content Blocking') }}"
-
-{% if l10n_has_tag('remove_fastblock_1499589') %}
-data-panel1-text="{{ _('When you see the shield, Firefox is blocking parts of the page that can track you online.') }}"
-{% else %}
-data-panel1-text="{{ _('When you see the shield, Firefox is blocking parts of the page that can slow your browsing or track you online.') }}"
-{% endif %}
-
-data-panel1-step="{{ _('1 of 3') }}"
-data-panel1-button="{{ _('Next') }}"
-data-panel3-title="{{ _('Turn off blocking for trusted sites') }}"
-
-{% if l10n_has_tag('content_blocking_65_1511420') %}
-data-panel3-text="{{ _('If content blocking prevents you from using a site you trust, select “Turn off Blocking Temporarily.”')|forceescape }}"
-{% else %}
-data-panel3-text="{{ _('If content blocking prevents you from using a site you trust, you can select “Disable Blocking Temporarily” in this panel.')|forceescape }}"
-{% endif %}
-
-{% if l10n_has_tag('content_blocking_65_1511420') %}
-data-panel3-text-new-tab="{{ _('If content blocking prevents you from using a site you trust, select “Turn off Blocking for This Site.”')|forceescape }}"
-{% else %}
-data-panel3-text-new-tab="{{ _('If content blocking prevents you from using a site you trust, you can select “Disable Blocking For This Site” in this panel.')|forceescape }}"
-{% endif %}
-
-data-panel3-title-alt="{{ _('Want to make changes?') }}"
-
-{% if l10n_has_tag('content_blocking_65_1511420') %}
-data-panel3-text-alt="{{ _('To restore content blocking for a site, select “Turn on Blocking for This Site.“')|forceescape }}"
-{% else %}
-data-panel3-text-alt="{{ _('To restore content blocking for a site, select “Enable Blocking For This Site“ in the Control Center panel.')|forceescape }}"
-{% endif %}
-
-data-panel3-step="{{ _('3 of 3') }}"
-data-panel3-button="{{ _('Got it!') }}"
-{% endblock %}
-
-{# Google Analytics should never be enabled on this page! #}
-{% block google_analytics %}{% endblock %}
-
-{# Stub Attribution data should never be collected on this page! #}
-{% block stub_attribution %}{% endblock %}
-
-{% block site_css %}
-  {{ css_bundle('tracking-protection-tour') }}
```

Comment:
Please remove this bundle too https://github.com/mozilla/bedrock/blob/master/media/static-bundles.json

You can remove this file as well: https://github.com/mozilla/bedrock/blob/master/media/css/firefox/tracking-protection-tour.less